docs/UserGuide.md

```diff
@@ -378,9 +378,52 @@ Format: `delnote INDEX`
 
 Example:
 
-`view 1` shows you the detailed view of  the contact at index 1, then using `delnote 2` will delete the second note in the 
+`view 1` shows you the detailed view of  the contact at index 1, then using `delnote 2` will delete the second note in the
 notes list of the contact
 
+### Add images : `addimg`
```

Comment:
Let's standardise using the continuous form in section headings. e.g. Adding Images 😃